### PR TITLE
test(bigtable): add UUID to avoid name collisions during test runs

### LINKED ISSUE #5145: bigtable: TestIntegration_BackupIAM failed
Note: #4165 was also for this test, but it was closed more than 10 days ago. So, I didn't mark it flaky.

----

commit: 33c0f63e0e0ce69d9ef6e57b04d1b8cc10ed2b78
buildURL: [Build Status](https://source.cloud.google.com/results/invocations/2f75dccc-dff9-494f-8d4c-8695109ab683), [Sponge](http://sponge2/2f75dccc-dff9-494f-8d4c-8695109ab683)
status: failed
<details><summary>Test output</summary><br><pre>integration_test.go:1410: Creating backup: rpc error: code = AlreadyExists desc = Backup backup already exists.</pre></details>

### PR DESCRIPTION
**Pull request by telpirion**
Fixes #5145.

### CODE REVIEW

File: bigtable/integration_test.go [hunk 1]
```diff
@@ -1400,8 +1401,10 @@ func TestIntegration_BackupIAM(t *testing.T) {
 	if err := adminClient.CreateTable(ctx, table); err != nil {
 		t.Fatalf("Creating table: %v", err)
 	}
+
 	// Create backup.
-	backup := "backup"
+	backupUUID := uuid.New()
```
Comment (codyoss) [inline bigtable/integration_test.go:1406]: Other manuals make use of https://github.com/googleapis/google-cloud-go/blob/main/internal/uid/uid.go#L54. You can avoid a dependency if you take this approach too 😄 .

**Review by codyoss — COMMENTED** [review]
Also, I tweaker your title. I would use test and you always want to provide a scope related to the directory path the change is made for release-please. Since this is just a test change a test scope is good enough and that way it will not show up in the release notes. Make sure to keep the new title when squashing.

File: bigtable/integration_test.go (hunk 1 above)
Comment (telpirion) [reply bigtable/integration_test.go]: Good call! Done.

**Review by codyoss — APPROVED** [review]

**Review by dmahugh — APPROVED** [review]
LGTM

**Pull request merged**